sdv/metadata/multi_table.py

```diff
@@ -180,9 +180,19 @@ def _validate_relationship_does_not_exist(
                 and relationship['child_table_name'] == child_table_name
                 and relationship['child_foreign_key'] == child_foreign_key
             )
+            foreign_key_already_exists = (
```

Comment:
minor: can we call this `foreign_key_used_in_other_relationship` or `foreign_key_already_used`?